### DISCUSSION TITLE
Alpha Day 1

### DISCUSSION THREAD

**Discussion by Alsamer98** [Alsamer-s-DDLC-Mods-and-Submods/Tales-of-Literature#7] [Ideas]
Post your Ideas here

**Comment by Hosna1100** [comment]
I don't have idea! I want to join you as I hope you have seen some of my DDLC mods in my own account that I worked alone for them. maybe...me and you can team up with each other (join me or I'll join you, if you want~)
here's a new DDLC mod idea of mine, as you wanted:
**"The Forgotten Room" Mod**:
    - The literature club discovers an abandoned room within the school building. It's locked, and no one remembers its existence.
    - As the MC investigates, they find old diaries, faded photographs, and unsettling drawings. The room's history intertwines with the girls' lives, revealing tragic events.
    - Monika, who has vague memories of the room, tries to suppress its influence. But the room's malevolence seeps into reality.

**Reply by Alsamer98** [reply]
Haha its kinda funny how your Reply came at a time like this tbh, I stopped working on my mod for a long time and started some weeks ago again, didnt think someone would play an Alpha thats 1 Year old. about your question, yeah sure we can try doing something together.

**Reply by Hosna1100** [reply]
hmm....I still enjoy any mods! (if their not write things like: btw) although, I'm saying if we can create something together because as you saw, I'm alone and you are too so, why won't we team up? it'll be great and I'll help you in mod_assets. see the repository named Doki Doki Mirror World to see how I edited Dokis sprites. (I think we can continue\create more mods together!)

**Reply by Alsamer98** [reply]
Yeah sure seems fine we can work on smt together.

**Reply by Hosna1100** [reply]
what is smt?

**Reply by Alsamer98** [reply]
Something --> I meant a  mod with that eheh...

**Reply by Hosna1100** [reply]
hmmm....sure! 
_I hope you've seen the idea that I've written for you above (that is named The Forgotten Room)_

**Reply by Hosna1100** [reply]
do you want another mod idea? because I have a GREAT idea!

**Reply by Alsamer98** [reply]
Sure tell me everything you've got

**Reply by Hosna1100** [reply]
let's make a DDLC mod that girls go to MC's house for having fun but suddenly, MC suggests them to play Sonic.exe and once the game starts, Dokis get teleported into that game and try to survive and get out of MC's computer. 
or maybe we can create our DDLC mod similar to that cool V2 Sonic.exe FNF mod.

**Reply by Alsamer98** [reply]
Ummm what now...the first Idea sound kinda intresting but why Sonic?

**Reply by Hosna1100** [reply]
have you seen Reddit DDLC community? they seemed to be eager to have a mod that has Sonic.exe and it seems cool I also know a visual novel that has Sonic.exe in it:
[https://vndb.org/v48841](url)

**Reply by Hosna1100** [reply]
Ahem! where are you? why are you absent there? I'm waiting for your reply! _or...maybe you're not a fast-reader like me._

**Reply by Alsamer98** [reply]
Sorryy I had to do things yesterday and didnt saw your messages.

**Reply by Hosna1100** [reply]
so did you played the game link I gave you? I meant, that Visual Novel of Sonic.exe.

**Reply by Hosna1100** [reply]
m "Can you hear me? then, did you even read any of my massages\replies??"

**Reply by Alsamer98** [reply]
Yes I read all of your replies, im just not always online

**Reply by Hosna1100** [reply]
I already created the logo, and I've done a little bit of script, I'll give you the google drive link, continue the script by your own imagination and when you done, give me the completed script to edit it and fix errors or grammar problems, you can add any CG you want! I'll draw it because I can use AI for it, now I just have to look for an Sonic.exe artist to draw us Sonic.exe sprites.
by the way, here's the link of script-ch0.rpy: [https://drive.google.com/file/d/1Z9o8RMSvmX95SiYcmIIbx9vgcBD-GwOF/view?usp=drive_link](url)
do you have Reddit account? maybe we can chat more with each other in there! (as nobody will know our plan in Reddit)
and PLEASE DON'T TELL ANYONE ABOUT OUR PLAN!

**Reply by Hosna1100** [reply]
Ahem!

**Reply by Hosna1100** [reply]
Bro, where are you? It's been years since we've talked!
